src/main/webapp/config.css

```diff
@@ -0,0 +1,84 @@
+.hidden {
+    display: none;
+}
+
+.text-align-right {
+    text-align: right;
+}
+
+.or-separator {
+    font-weight: bold;
+}
+
+.margin-bottom-2em {
+    margin-bottom: 2em;
+}
+
+.margin-right-1em {
+    margin-right: 1em;
+}
+
+.site-table {
+    table-layout: fixed;
+    width: 100%;
+}
+
+.col-25 {
+    width: 25%;
+}
+
+.col-35 {
+    width: 35%;
+}
+
+.col-10 {
+    width: 10%;
+}
+
+.sticky-save-footer {
+    position: sticky;
+    bottom: -1px;
+    margin-top: 2em;
+    margin-left: -2em;
+    width: 100vw;
+    z-index: 998;
+    background: #fff;
+    padding: 1em 0em 1em 2em;
+}
+
+.sticky-save-footer button {
+    min-width: 5.625rem;
+}
+
+input[type=button].validate-regex-btn {
+    float: right;
+    margin-top: 0.5em;
+}
+
+/* Override Jelly core layout offset issue */
+#page-body #main-panel {
+    max-width: 80vw!important;
+}
+
+tr.site-data-row.selected td {
```

Comment:
sometimes, !important is needed

Reply:
Agreed but as a nit: spacing 😄

Reply:
Agreed but I have a nit: space before the !